src/features/courses/list-of-courses/list-of-courses.js

```diff
@@ -53,7 +53,7 @@ export const ListOfCourses = () => {
         <tr key={course.id} onClick={(event) => courseDetails(course.id)} className={styles['table-row']} data-student-id={course.id}>
           <td className="text-center">{course.id}</td>
           <td>{course.name}</td>
-          {currentUser.role !== 2 &&
+          {currentUser.role !== 2 && currentUser.role !== 1 &&
```

Comment:
role 2 - for what users?
Maybe we can check with  ===, not !== ?

Reply:
As I remember Mentors have role 2;
So your suggestion makes it to opposite roles with '===', right?

Reply:
I thought, and I guess that probably was a mistake by previous developers, they wanted to make it for 'students' role but did it for 'mentors'. 
So now I'm gonna change it just for students with '===' because mentors should have access to editing, right?